python/lib/modeldata/dmod/test/test_bmi_init_config_auto_generator.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import pandas as pd
+import geopandas as gpd
+
+from pathlib import Path
+from ..modeldata.data.bmi_init_config_auto_generator import BmiInitConfigAutoGenerator, NgenRealization
+from typing import Dict
+from ngen.init_config.serializer_deserializer import (IniSerializerDeserializer, JsonSerializerDeserializer,
+                                                      NamelistSerializerDeserializer, YamlSerializerDeserializer)
+
+
+@unittest.skip
+class TestBmiInitConfigAutoGenerator(unittest.TestCase):
+
+    @staticmethod
+    def _remove_dir(dir_path: Path):
```

Comment:
The only tangible benefit to not using it is avoiding another import.  Not nothing, but close.  But so was the effort to implement `_remove_dir`.  I do also like to avoid mixing file-related operations between `pathlib` and alternatives, but that's a personal style thing.